data/inputs/gri30_ion.cti

```diff
@@ -0,0 +1,2192 @@
+#
```

Comment:
It is still a problem to estimate the diffusion coefficient. The charge species interaction cannot use L-J model, so I need to write a new one and import different data.